### PR TITLE
Add python-picard

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hoechenberger** [comment]
The `bioconda` package with the same name is a _different_ software.

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes This PR is ready for review. Please note that this package's `setup.py` makes use of `numpy.distutils.core.setup()`, hence I had to list `numpy` as a `host` requirement.

**Comment by cbrueffer** [comment]
As noted by the linter, there's a very well known bioinformatics package in bioconda with the same name.  I'd suggest renaming this package to `py-picard` or similar to avoid a naming conflict.

File: recipes/picard/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% set name = "picard" %}
+{% set pypi_name = "python-picard" %}
+{% set version = "0.4" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (chrisburr) [inline recipes/picard/meta.yaml]: What do you think about making this the same as the  PyPI name? It seems more intuitive to me but I know nothing about either package.

**Comment by chrisburr** [comment]
> The `bioconda` package with the same name is a _different_ software.

The problem is that `bioconda` uses `conda-forge` as a higher priority channel. Adding it with the current name will break installation and updating for any users of that package.

File: recipes/picard/meta.yaml (hunk 1 above)
Comment (hoechenberger) [reply recipes/picard/meta.yaml]: I don't really have a strong opinion on that; I've just changed it in aefb43a

**Comment by hoechenberger** [comment]
> The problem is that `bioconda` uses `conda-forge` as a higher priority channel. Adding it with the current name will break installation and updating for any users of that package.

Okay cool, so that's a strong argument for picking a different name then!

**Comment by chrisburr** [comment]
Thanks for the PR! 🍰

**Comment by hoechenberger** [comment]
Thanks for your work :)

**Pull request merged**